R/tt_export.R

```diff
@@ -942,7 +964,11 @@ tt_to_flextable <- function(tt,
   flx <- flextable::width(flx, width = final_cwidths) # xxx to fix
 
   if (!is.null(theme)) {
-    flx <- theme(flx)
+    flx <- theme(
+      flx,
+      tbl_ncol_body = NCOL(tt),
+      tbl_row_class = make_row_df(tt)$node_class # These are ignored if not in the theme
+    )
```

Comment:
this is why theme should always have ...

Reply:
what if we introduce a default, and dummy theme

Reply:
you mean instead of null?